public/app/features/logs/components/getLogRowStyles.ts

```diff
@@ -89,6 +89,10 @@ export const getLogRowStyles = memoizeOne((theme: GrafanaTheme2) => {
           z-index: 1;
         }
 
+        .log-row-menu-visible {
```

Comment:
Where do we use `log-row-menu-visible`? I couldn't find it being used anywhere.

Reply:
Good 👀 . Removed in 5134a52efd473b23946e0f454a4fd3ea99d9203e

Reply:
I am learning from the best 👀